### DISCUSSION TITLE
Welcome to Movie-Project Discussions!

### DISCUSSION THREAD

**Discussion by NickoBrown** [NickoBrown/Movie-Project#1] [Discussions]
Discussion page for RAD Movie project

**Comment by NickoBrown** [comment]
Test comment for discussions

**Reply by kanicc98** [reply]
Hey Nick,
I have finish all the coding required for this sprint.
Just need to touch up on some formatting in class!
Do you need any help with the documentation part?

**Reply by NickoBrown** [reply]
Hi Duy,
Apologies for the late reply, I did not think to check here 😐
I have finished the software review plan, testing plan and performance report, so I think that is all the docs we need!
I will run the details of the review plan by you in class and we can change them if needed.
-N

**Comment by kanicc98** [comment]
I have added new table and a membership php page. I will get all the coding finish on wednesday

**Comment by kanicc98** [comment]
Hey Nick,
I will get my coding part done by wednesday and ill do the testing documents in class on wednesday
